src/stream/src/executor/mod.rs

```diff
@@ -307,6 +308,7 @@ pub enum Mutation {
     Resume,
     Throttle(HashMap<ActorId, Option<u32>>),
     AddAndUpdate(AddMutation, UpdateMutation),
+    AlterSinkConfig(HashMap<ActorId, HashMap<String, String>>),
```

Comment:
The sink executor can match the alter sink config by its `sink_id` rather than `actor_id`. Therefore, it will be better to make it `AlterSinkConfig(SinkId, HashMap<String, String>)`.